lib/modules/auth/infra/firebase_interface.dart

```diff
@@ -54,10 +54,10 @@ class FirebaseInterface {
           .signInWithEmailAndPassword(email: email, password: password);
       return null;
     } on FirebaseAuthException catch (e) {
-      if (e.code == 'user-not-found') {
-        print('No user found for that email.');
-      } else if (e.code == 'wrong-password') {
-        print('Wrong password provided for that user.');
+      if (e.code == 'Usuário não encontrado') {
+        print('Nenhum usuário encontrado para esse email');
+      } else if (e.code == 'Senha incorreta') {
+        print('Senha incorreta para esse usuário');
```

Comment:
Essa parte não está errada, mas é interessante retornar a mesma mensagem nos dois casos pra dificultar um ataque. Se o atacante souber que o nome/email do usuário está correto ele pode tentar um ataque de força bruta na senha pra tentar burlar a autenticação e descobrir a senha do usuário. Com uma mesma mensagem ele não vai saber se precisa alterar a senha ou o noe de usuário e dificulta um pouco o ataque